elf_reader.go

```diff
@@ -1379,6 +1415,174 @@ func (ec *elfCode) loadKsymsSection() error {
 	return nil
 }
 
+// loadStructOpsMapsFromSections creates StructOps MapSpecs from DataSec sections
+// ".struct_ops" and ".struct_ops.link" found in the object BTF.
+func (ec *elfCode) loadStructOpsMaps() error {
```

Comment:
This should be merged with associateStructOpsRelocs unless there is a reason why they need to be separate which I don't understand.

Reply:
Merged